File: src/rider/main/kotlin/com/jetbrains/rider/plugins/testlinker/GotoLinkedTypesAction.kt
package com.jetbrains.rider.plugins.testlinker

import com.jetbrains.rider.actions.base.RiderAnAction
import icons.ReSharperIcons.UnitTesting

class GotoLinkedTypesAction : RiderAnAction(
        "GotoLinkedTypesAction",
        "Goto Linked Types",
        null,
        UnitTesting.TestFixtureToolWindow)

File: src/dotnet/ReSharperPlugin.TestLinker/Actions/GotoLinkedTypesAction.cs
using JetBrains.Application.UI.ActionsRevised.Menu;
using JetBrains.ReSharper.Feature.Services.Navigation.ContextNavigation;
using ReSharperPlugin.TestLinker.Navigation;

namespace ReSharperPlugin.TestLinker.Actions
{
    [Action(Id, "Goto Linked Types", Id = 170, IdeaShortcuts = new[] {"Shift+Control+I"}, VsShortcuts = new[] {"Shift+Control+I"})]
    public class GotoLinkedTypesAction
        : ContextNavigationActionBase<LinkedTypesNavigationProvider>
    {
        public const string Id = nameof(GotoLinkedTypesAction);
    }
}